assets/search/search.controller.js

```diff
@@ -136,18 +141,18 @@ module.exports = function ($scope, $timeout, UserFactory) {
       url += "&user=" + vm.input.user;
     }
     url += "&skip=" + skip;
-
-    var searchedFor = url;
+    vm.done = false;
     io.socket.get(url, function (data, res) {
-      if (res.statusCode === 200 && searchedFor === url) {
+      if (res.statusCode === 200) {
```

Comment:
I see, that makes sense. But in that case I think we need to make searchedFor a persistent variable, because otherwise the url gets compared to a separate version of searchedFor.